Road/notebooks/data_cleaning.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            text = DC.remove_multimedia(text)
</pre><p>When the message contains multimedia, the message is empty, then you should drop it directly because it is an empty message.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>